includes/KSES.php

```diff
@@ -564,13 +564,58 @@ public function filter_kses_allowed_html( $allowed_tags ) {
 			],
 		];
 
-		$allowed_tags = array_merge( $allowed_tags, $story_components );
+		$allowed_tags = $this->array_merge_recursive_distinct( $allowed_tags, $story_components );
 
 		$allowed_tags = array_map( [ $this, 'add_global_attributes' ], $allowed_tags );
 
 		return $allowed_tags;
 	}
 
+	/**
+	 * Recursively merge a variable number of arrays, using the left array as base,
+	 * giving priority to the right array.
+	 *
+	 * Difference with native array_merge_recursive():
+	 * array_merge_recursive converts values with duplicate keys to arrays rather than
+	 * overwriting the value in the first array with the duplicate value in the second array.
+	 *
+	 * array_merge_recursive_distinct does not change the data types of the values in the arrays.
+	 * Matching keys' values in the second array overwrite those in the first array, as is the
+	 * case with array_merge.
+	 *
+	 * Freely based on information found on http://www.php.net/manual/en/function.array-merge-recursive.php
+	 *
```

Comment:
Not sure what version to add here, as would go out in 1.5 or 1.4.1?

Reply:
1.5

Reply:
Fixed.